README.md

```diff
@@ -168,6 +168,10 @@ Collected JSON objects are wrapped into the protobuf [structure](https://github.
 The `EHubGeneral` function listens to `alertlogic-log` which is created during [collector setup](#deploy-with-the-custom-arm-template-in-an-azure-subscription). The `alertlogicloghub` event hub can be used for integration with, for example, [diagnostic logs](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/diagnostic-logs-stream-event-hubs) or [Azure AD logs](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/tutorial-azure-monitor-stream-logs-to-event-hub).
 Collected JSON objects are wrapped into the protobuf [structure](https://github.com/alertlogic/al-collector-js/blob/master/proto/common_proto.piqi.proto) and then are forwarded to the Alert Logic Ingestion service.
 
+## DLBlob Function
```

Comment:
I think an additional paragraph describing where this comes from would be helpful. Something to the effect:

Both EHub collector functions (EHubActivityLogs, EHubGeneral) may not be able to process incoming messages. If that's the case, those messages will be saved as blobs to the `alertlogic-dl` container so that collection can be retried at a later time.